docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+UI framework has its own specific bindings library. For example, the [React-Redux library](https://react-redux.js.org) allows your React components to read values from a Redux store,
+and tell the store to calculate an update to the state inside.
+
+### What is Redux Toolkit?
+
+Redux itself is small and unopinionated. We also have a separate addon package called [Redux Toolkit](https://redux-toolkit.js.org), which includes some opinionated defaults
+that help you use Redux more effectively, and includes packages that we think are essential to building a Redux app.
+It's our official recommended approach for writing Redux logic. As with the Redux core, you can use Redux Toolkit with any UI layer.
+
+## Goals
+
+We're going to create a small application with React and Redux together. The app will be a simple employee database, and we'll learn:
+
+- How to set up Redux with React
+- The data flow lifecycle of a Redux app
+- How to write Redux logic that updates data, and shows the updated data in the UI
+- How to create, update, view, and delete users (employees) from the system
+- How to make API calls for each of the above actions
+
+## Setup and Installation
+
+### Create a React Project
+
+We'll start by creating an empty React project. You can do this in one of two ways: creating a local project on your own computer with the
+Create-React-App tool, or creating a project in the CodeSandbox web app.
+
+#### Create a Local Project with Create-React-App
+
+[Create-React-App](https://create-react-app.dev/) is the official tool for creating new React projects.
+
+Create a new React project named `redux-employee-example` by running this command:
+
+```bash
+npx create-react-app redux-employee-example
+```
+
+Then, install the required Redux packages, using either the NPM or Yarn package manager:
+
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: 'NPM', value: 'npm', },
+    { label: 'Yarn', value: 'yarn', }
+  ]
+}>
+
+<TabItem value="npm">
+
+```bash
+npm install @reduxjs/toolkit react-redux
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add @reduxjs/toolkit react-redux
+```
+
+</TabItem>
+
+</Tabs>
+
+#### Create an Online Project with CodeSandbox
+
+[CodeSandbox.io](https://codesandbox.io) is an online IDE that lets you create entire web applications in your browser, without needing to
+install anything locally.
+
+Create a new React project sandbox by browsing to the [CodeSandbox "Create Sandbox" page](https://codesandbox.io/s/) and selecting the official React template.
+
+Once the sandbox has been created, click the "Add Dependency" button on the left, search for `@reduxjs/toolkit`, add it, and then do the same for `react-redux`.
+
+### Create a Redux Store
+
+The React project starts with a single `<App>` component that shows some "hello" text on the page. Our first goal is to create a Redux store instance
+that holds some data inside, and update the `<App>` component to read that data and show it instead.
+
```

Comment:
I think it'd be good to show a full working React example before adding the Redux store. Maybe something with state.

Presumably, landing people on this page have used React a little. Ideally they have, anyway. Starting with a React example and "refactoring it to Redux" would:

- help a person who already knows React feel grounded, adding new knowledge on top of old
- help a person who has never seen React realize that they should go learn that first (provide a link to official tutorial) and that Redux builds on top of React